materials/functions-R.md

```diff
@@ -56,7 +56,18 @@ calc_shrub_vol(0.8, 1.6, 2.0)
 shrub_vol <- calc_shrub_vol(0.8, 1.6, 2.0)
 ```
 
+<<<<<<< Updated upstream
```

Comment:
Is this line and lines 61 and 70 supposed to be here? I don't really understand their purpose, this looks sort of like a merge conflict.